usaspending_api/etl/management/commands/load_table_from_delta.py

```diff
@@ -239,6 +242,19 @@ def handle(self, *args, **options):
                     cursor.execute(create_temp_sql)
                     logger.info(f"{temp_table} created.")
 
+                    # If there are vectors, add the triggers that will populate them based on other calls
+                    logger.info(f"Adding tsvector triggers for the following columns: {list(tsvectors.keys())}")
+                    for tsvector_name, derived_from_cols in tsvectors.items():
+                        derived_from_cols_str = ", ".join(derived_from_cols)
+                        tsvector_trigger_sql = f"""
+                            CREATE TRIGGER tsvector_update_{tsvector_name} BEFORE INSERT OR UPDATE
```

Comment:
Any way to do "if not exists" for triggers? Given kerfuffle and error states with swaping, and copying-back, we may find some db objects already existing.

Reply:
Postgres seems to have something like that in Postgres 14. In the meantime, I'll just drop it if it does exist to be safe.

https://stackoverflow.com/questions/35927365/create-or-replace-trigger-postgres